src/OrchardCore.Modules/OrchardCore.Placements/Views/Admin/Edit.cshtml

```diff
@@ -0,0 +1,70 @@
+@model EditShapePlacementViewModel
+@{
+    var returnUrl = ViewData["returnUrl"]?.ToString();
+}
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-addon-display-autorefresh" at="Foot"></script>
+<script asp-name="codemirror-addon-selection-active-line" at="Foot"></script>
+<script asp-name="codemirror-mode-javascript" at="Foot"></script>
+
+<h1>@RenderTitleSegments(T["Edit Placement"])</h1>
```

Comment:
This should be `Create` if you're creating right?

Because this is a shared view for both create and edit.